src/styles/card.scss

```diff
@@ -0,0 +1,115 @@
+@import './variables';
+@import './mixins';
+
+.container {
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 200px);
+  gap: 48px 48px;
+  justify-content: center;
+
+  @include two-columns {
+    grid-template-columns: repeat(auto-fit, 200px);
+  }
+
+  @include three-columns {
+    grid-template-columns: repeat(auto-fit, 200px);
+  }
+
+  @include four-columns {
+    grid-template-columns: repeat(auto-fit, 200px);
```

Comment:
Nice use of SASS features, you could also create it with plain CSS but this way looks cleaner, good job